### DISCUSSION TITLE
İBAX nedir ne amaçla kullanılacak ( konu içinde fiyat ne olur vs gibi girmeyelim )

### DISCUSSION THREAD

**Discussion by ahmederoni** [IBAX-io/go-ibax#1314] [Soru Cevap - Türkçe]
Bu başlık altında ibax ın nerde ne amaçla ve kimler tarafından kullanılacagı hakkında bilgi paylaşımı yapalım

**Comment by Yusufknk** [comment]
Bu bilgilerin hepsini ibax sayfasını detaylı inceleyerek ulaşabilirsiniz.
